dbms/src/Flash/Coprocessor/JoinInterpreterHelper.cpp

```diff
@@ -374,5 +375,31 @@ std::tuple<ExpressionActionsPtr, Names, Names, String> prepareJoin(
     dag_analyzer.appendJoinKeyAndJoinFilters(chain, keys, join_key_types, key_names, original_key_names, left, is_right_out_join, filters, filter_column_name);
     return {chain.getLastActions(), std::move(key_names), std::move(original_key_names), std::move(filter_column_name)};
 }
+
+std::vector<RuntimeFilterPtr> TiFlashJoin::genRuntimeFilterList(const Context & context,
+                                                                const Block & input_header,
+                                                                const LoggerPtr & log)
+{
```

Comment:
Skip generate runtime filter for the join that is not support runtime filter? like cross join and nullaware join.

Reply:
Return if runtime filter list is empty

Reply:
I mean double check the join type and return empty list if TiFlash does not support runtime filter for current join type. (According to previous experience, TiFlash should always assume TiDB may generate a plan that is actually not supported by TiFlash yet.)

Reply:
I don't think this should be repeated in tiflash to check the logic of the join type.
This results in the same logic being placed in different places. If the rear side is changed, the other side will definitely be missed, which is not a good design.